server/flow.go

```diff
@@ -509,10 +509,6 @@ func (fm *FlowManager) submitOAuthConfig(f *flow.Flow, submitted map[string]inte
 
 	clientID = strings.TrimSpace(clientID)
 
-	if len(clientID) != 64 {
-		errorList["client_id"] = "Client ID should be 64 characters long"
-	}
-
```

Comment:
Alternatively we could check `> 1` or `>= 64`

Reply:
Let's check for `>= 64`